src/System.Net.HttpListener/src/System/Net/HttpListenerRequest.cs

```diff
@@ -232,6 +272,14 @@ public Uri UrlReferrer
 
         public Version ProtocolVersion => _version;
 
+        public Task<X509Certificate2> GetClientCertificateAsync()
```

Comment:
This is just a random addition that snuck in to this PR. I was trying to diagnose why the following:
```
IAsyncResult res = request.BeginGetClientCertificate(null, null);
var cert = request.EndGetClientCertificate(res);
```

hangs forever with the managed implementation